### PR TITLE
MVP Done

### PR DESCRIPTION
**Pull request by jthicks91**
MVP Done. Half of the project is done for MVP. Will continue to work on it to complete it fully tomorrow.

### CODE REVIEW

File: src/03_modules.py [hunk 1]
```diff
@@ -10,25 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-for arg in sys.argv:
-    print(arg)
+print("The CLI Argument is: " + str(sys.argv))
```
Comment (jeffyjkang) [inline src/03_modules.py:13]: you had this correct, before, you want this in a loop, instructions say one per line

File: src/03_modules.py [hunk 2]
```diff
@@ -10,25 +10,30 @@
...
+pid = os.getpid()
+print (pid)
 
 # Print the current working directory (cwd):
 # YOUR CODE HERE
+cwd = os.getcwd()
+print (cwd)
 
 # Print out your machine's login name
 # YOUR CODE HERE
```
Comment (jeffyjkang) [inline src/03_modules.py:37]: im getting os has no attribute uname, i think you can just do os.getlogin()

**Review by jeffyjkang — APPROVED** [review]
great work on intro-python-i mvp

File: src/03_modules.py (hunk 1 above)
Comment (jthicks91) [reply src/03_modules.py:13]: got it, thanks! Instructions were a bit confusing

File: src/03_modules.py (hunk 2 above)
Comment (jthicks91) [reply src/03_modules.py:37]: got it, will change.

**Pull request merged**